src/js/views/responses/item.js

```diff
@@ -126,6 +126,11 @@ function($, _, Backbone, events, settings, util, api, Responses, template) {
         } else if (question.type === 'text') {
```

Comment:
Do we even need this block? Answered text questions are picked up by the block above. Is it used for legacy response formats?